http/src/main/resources/logback.xml

```diff
@@ -27,7 +32,12 @@
     <appender name="SYSLOG" class="ch.qos.logback.classic.net.SyslogAppender">
         <syslogHost>127.0.0.1</syslogHost>
         <facility>AUDIT</facility>
-        <suffixPattern>[%level] [%d{HH:mm:ss.SSS}] [%thread] %logger{36} - %msg%n</suffixPattern>
+<!--        <suffixPattern>[%level] [%d{HH:mm:ss.SSS}] [%thread] %logger{36} - %msg%n</suffixPattern>-->
```

Comment:
is this commenting out intentional?

Reply:
yea, I want the encoder below to take effect

Reply:
Hmm.. I want to be careful of this. Do any downstream systems depend on our log format? The LogStash -> Data Warehouse stuff might be depending on this.

Reply:
I'd recommend not changing the Leo log format in this PR